### PR TITLE
Add a mouse over pixel selection tool

### PR DESCRIPTION
**Pull request by Cadair**
This dosen't seem to work, i.e. the button is not there, I can't figure out how to make it register with the image viewer.

### CODE REVIEW

**Comment by astrofrog** [comment]
@Cadair - add it to the list of tools on line 64 in the ``image/data_viewer.py`` file

**Comment by astrofrog** [comment]
@Cadair - thanks! Can you add an entry to the changelog under v0.13.0? (order doesn't matter but maybe near the bottom to avoid conflict with another PR which adds an entry near the top)

**Comment by astrofrog** [comment]
Currently this doesn't require the user to click at the start, merely hovering over the image starts the selection. I would prefer a click to be required to start the selection, so maybe _on_move should default to False?

File: glue/viewers/image/qt/data_viewer.py [hunk 1]
```diff
@@ -63,7 +64,7 @@ class ImageViewer(MatplotlibDataViewer):
     tools = ['select:rectangle', 'select:xrange',
              'select:yrange', 'select:circle',
              'select:polygon', 'image:contrast_bias',
-             'save:python']
+             'image:point_selection', 'save:python']
```
Comment (astrofrog) [inline glue/viewers/image/qt/data_viewer.py]: Can you put it just *before* contrast_bias?

Comment (Cadair) [reply glue/viewers/image/qt/data_viewer.py]: sure! How are you ordering them? they are alphabetical now right?

**Comment by Cadair** [comment]
Thanks!

**Pull request merged**